conmon-rs/common/proto/conmon.capnp

```diff
@@ -19,4 +23,33 @@ interface Conmon {
     }
 
     createContainer @1 (request: CreateContainerRequest) -> (response: CreateContainerResponse);
-}
+
+    ###############################################
+    # ExecContainer
+    struct ExecContainerRequest {
+        id @0 :Text;
+        timeout  @1 :Int32;
+        cmd @2 :List(Text);
+
+        # todo
+        #   stdin
+        #   stdout
+        #   stderr
+        #   tty
+    }
+
+    struct ExecContainerHandle {
+    }
+
+    enum ExecChunkType {
+        stdout @0;
+        stderr @1;
+    }
+
+    interface ExecCallback {
+        sendChunk @0 (type :ExecChunkType, chunk :Data);
+        done @1 ();
+    }
```

Comment:
I was thinking if it would make sense to merge `done` into `sendChunk` and pass the exit code as another parameter. But I guess having two methods is fine as well.